tests/chains/ethereum/contract/migrations/2_deploy_contracts.js

```diff
@@ -0,0 +1,35 @@
+const IBFT2Client = artifacts.require("IBFT2Client");
```

Comment:
You can remove IBFT2Client.

Reply:
I did this by deleting IBFT2Client.
When I deleted it, the address of SimpleToken changed.
When the address of SimpleToken was changed, it mismatched with the address defined here, and I could not get the correct baseDenom.

https://github.com/hyperledger-labs/yui-ibc-solidity/blob/a1cf1354c0ad577ffcb775ced4e90c0c396adfa9/pkg/consts/contract.go#L10

https://github.com/dongrie/yui-relayer-build/blob/v0.3/tests/chains/ethereum/scripts/cmd/balance/main.go#L49

Reply:
`yui-ibc-solidity/pkg/consts/contract.go` is only for e2e/integration testing of yui-ibc-solidity.
These CLI tools shouldn't refer to these constant variables.

Instead, the tools should get these values from the command line arguments.

Reply:
https://github.com/dongrie/yui-relayer-build/blob/v0.3/tests/chains/ethereum/scripts/cmd/helper/contract.go
Created own contract.go file

Or should I put these addresses in path.json as well?